packages/webpack-config/src/webpack/dev/tasks/loadSourceMaps/module.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * add source-map of any third party dependency to the output source-map
+ */
+export const moduleLoadSourceMaps = {
+  module: {
+    rules: [{
+      // assuming third party dependencies such as @kluntje have only .js or .mjs extensions
+      test: /\.(js|mjs)$/,
+      use: [
+        "source-map-loader",
+      ]
+    }],
+  },
+};
```

Comment:
maybe we could also use this in legacy-bundle

Reply:
i thought probably the heavy debuging will always happen in some modern browsers. altough IE should have a good enough support for source maps but i'm not sure if it overwhelms IE when the source map is big. tbh i didn't measure the impact on the build time.
what do you think?

Reply:
i think there could be a use-case to debug a ie11 specific issue in ie11. but from my perspective we could also leave it as it is for now and add legacy-sourcemaps when we actually need them.